### PR TITLE
this commit involves navigation from article to its comment page; addC…

### PR DESCRIPTION
**Pull request by VarshaJacob**
…omment and voting  functionality not added

### CODE REVIEW

File: src/components/AddComment.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const AddComment = () => {
+    return (
+        <div className="AddComment">
```
Comment (lewispricey) [inline src/components/AddComment.jsx]: Is there a more semantic html tag you could use to signify this is an area for user input?

File: src/components/CommentList.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    },[article_id])
+
+
+
+    if (isLoading) {
+        return <p>Loading ...</p>
+    }
+
+    return (
+        <div className="CommentList">
```
Comment (lewispricey) [inline src/components/CommentList.jsx:28]: Could we find another HTML Tag to use to contain the list of comments that has more meaning to a screen reader?

File: src/components/CommentList.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            <ol>
```
Comment (lewispricey) [inline src/components/CommentList.jsx:29]: Really like the use of OL here seperating comments into individual LI's super clear you're rendering a list

File: src/components/CommentList.jsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+import { useParams } from "react-router-dom";
+import { getCommentsByArticle } from "../api";
+import AddComment from "./AddComment";
+
+
+const CommentList = () => {
+
+    const {article_id} = useParams()
+    const [commentList, setCommentList] = useState([])
+    const [isLoading,setIsLoading] = useState(true)
```
Comment (lewispricey) [inline src/components/CommentList.jsx:11]: Awesome to include the isLoading for the comments section, it's really important users know something is happening while on a slower connection.

**Review by lewispricey — CHANGES_REQUESTED** [review]
Great work so far Varsha! Realy like the simple but clear styling pattern 👍 

I've left a few comments on the code but overall its solid.

You're currently rendering comments as a seperate page/route, It may be better to have these comments rendering on the individual article page (maybe with a show/hide button) so the user has some context. A little like Youtube or Linked in comments with these apperaning below the related article.

Please have a think how you could render the comments on the individual articles and raise another PR.

File: src/components/AddComment.jsx (hunk 1 above)
Comment (VarshaJacob) [reply src/components/AddComment.jsx]: changed to form tag

File: src/components/CommentList.jsx (hunk 2 above)
Comment (VarshaJacob) [reply src/components/CommentList.jsx:28]: not sure which tag would be appropriate

**Review by leekli — APPROVED** [review]
Good work Varsha, thanks for making these amendments.

You are free to merge this branch.

**Pull request merged**